src/sentry/api/serializers/models/discoversavedquery.py

```diff
@@ -32,7 +33,9 @@ def serialize(self, obj, attrs, user, **kwargs):
             "version": obj.version or obj.query.get("version", 1),
             "dateCreated": obj.date_created,
             "dateUpdated": obj.date_updated,
-            "createdBy": six.text_type(obj.created_by_id) if obj.created_by_id else None,
+            "createdBy": serialize(obj.created_by, serializer=UserSerializer())
```

Comment:
@wmak Do we eager load this relationship or are we adding an N query?

Reply:
In the case of a post to the saved query endpoint, this is a `request.user`'s `SimpleLazyObject` so I think we add a query in that case.
For a get I think this was already loaded.

Reply:
The only prefetch_related I see on the GET is for projects.

Reply:
Ah you're right, i'll add a `select_related` to the query :+1: